### PR TITLE
#223

### PR DESCRIPTION
**Pull request by jmverges**

### CODE REVIEW

File: Classes/Backend/ContentSelector.php [hunk 1]
```diff
@@ -43,17 +43,36 @@ public function renderField(array &$parameters, &$parentObject) {
...
 		}
-		$select = '<div><select ' .
-			'style="background: #fff url(' . $selectedIcon . ') 5px 50% / 16px 16px no-repeat; padding-top: 2px; padding-left: 24px;" ' .
-			'name="' . htmlspecialchars($name) . '"  class="formField select" onchange="if (confirm(TBE_EDITOR.labels.onChangeAlert) && TBE_EDITOR.checkSubmit(-1)){ TBE_EDITOR.submitForm() };">' . LF;
+		if (\TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) < 7000000 ) {
+			$select = '<div><select ' .
+				'style="background: #fff url(' . $selectedIcon . ') 5px 50% / 16px 16px no-repeat; padding-top: 2px; padding-left: 24px;" ' .
+				'name="' . htmlspecialchars($name) . '"  class="formField select" onchange="if (confirm(TBE_EDITOR.labels.onChangeAlert) && TBE_EDITOR.checkSubmit(-1)){ TBE_EDITOR.submitForm() };">' . LF;
+		} else {
+			$select = '<div class="form-control-wrap"><div class="input-group">' .
```
Comment (xf-) [inline Classes/Backend/ContentSelector.php]: 2 input group classes.

**Comment by xf-** [comment]
Before it was ugly and this gets even more horrible. We should find a nicer template idea.
Does 6.2 has a problem with additional containers?

**Comment by NamelessCoder** [comment]
Solution coming momentarily which splits this class into two - one to be used in current versions and one for the legacy 6.2 rendering; with shared methods as needed. Will update here.

**Comment by NamelessCoder** [comment]
Applied in https://github.com/FluidTYPO3/fluidcontent/commit/f345f7fb0439d37d1ebfa589ce161f1ead8728d0

**Pull request closed without merging**